ods_ci/tests/Resources/Common.robot

```diff
@@ -477,7 +477,6 @@ Run And Watch Command
 Check Process Output and Status
   [Documentation]    Helper keyward for 'Run And Watch Command', to tail proccess and check its status
   [Arguments]    ${process_id}    ${process_log}    ${temp_log}
-  Log To Console    .    no_newline=true
```

Comment:
Why to remove this line ?
It is a good indication that something is going on ...